tests/foreman/cli/test_leapp_client.py

```diff
@@ -199,6 +200,20 @@ def precondition_check_upgrade_and_install_leapp_tool(custom_leapp_host):
     assert custom_leapp_host.run('yum install leapp-upgrade -y').status == 0
     if custom_leapp_host.run('needs-restarting -r').status == 1:
         custom_leapp_host.power_control(state='reboot', ensure=True)
+    if custom_leapp_host.os_version.major == 8:
+        # Inhibitor - Firewalld Configuration AllowZoneDrifting Is Unsupported
```

Comment:
Current fixture is just for install leapp tool, if you're using the same fixture to resolve inhibitors I'd suggest updating the fixture name or create a new fixture to resolve inhibitors

Reply:
I would prefer extending this fixture with added description to resolve inhibitors.

Reply:
fixture name looks fine. `precondition_check_upgrade` means we are checking the condition for upgrade.

Reply:
I mean something which says it fixes inhibitors, but its upto you